test/intg/rules.test.js

```diff
@@ -120,11 +120,36 @@ describe('Rule API', function() {
   async function createVictoriaBuild() {
     await initializeInfrastructure();
     const [p, v] = [theProperty, victoria];
-    const e = await helpers.createTestEnvironment(p.id, 'SouthernCross');
+    const e = await helpers.createTestEnvironment(
+      p.id,
+      'SouthernCross',
+      'Akamai'
+    );
     const r = await reactor.setEnvironmentRelationshipForLibrary(v.id, e.id);
     expect(r.data.id).toBe(e.id);
     expect(r.links.related).toContain(`/${v.id}/`);
-    return (await reactor.createBuild(v.id)).data;
+
+    const build = (await reactor.createBuild(v.id)).data;
+    const buildId = build.id;
+    expect(buildId).toMatch(helpers.idBL);
+
+    // wait for build to complete
+    const totalWait = 30000; // in milliseconds
+    const pollInterval = 2000; // in milliseconds
+    for (var i = 0; i < totalWait; i += pollInterval) {
+      await helpers.sleep(pollInterval);
+      var getBuildResponse = await reactor.getBuild(buildId);
+      reactor.logger.always(
+        `after ${i + pollInterval} milliseconds, ${buildId} is ${
+          getBuildResponse.data.attributes.status
+        }`
+      );
+      if (getBuildResponse.data.attributes.status !== 'pending') break;
+    }
```

Comment:
I haven't completely compared code, but it looks a lot like what I see in libraries.test.js. Should this be split out into a shared utility?

Reply:
I've pulled the "build library" behavior into a helper function, and now it's used everywhere library builds are needed.